### DISCUSSION TITLE
Kendine göndermeli doğrulama sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#224] [Zeka Sorusu] [answered]
Aşağıdaki seçeneklerden sadece biri doğruysa doğru olan seçenek hangisidir?

A ) E seçeneği doğrudur
B ) A seçeneği doğru değildir
C ) D seçeneği ve B seçeneği yanlıştır
D ) Hepsi doğrudur
E ) Hepsi yanlıştır

**Comment by FatihOnder81** [comment, marked as answer]
Cevap b şıkkıdır çünkü d şıkkı olamaz çünkü hepsi doğru diyor burada sadece bir tanesi doğrudur diyor a şıkkıda olamaz ordada başka bir şıkkın doğru olduğunu iddia ediyor a şıkkı yanlışsa b seçeneği mantıken doğru olur
